### PR TITLE
[ICCVID][MSRLE32][MSVIDC32] Add VfW codecs version info

### PR DESCRIPTION
**Pull request by katahiromz**
## Purpose

Based on KRosUser's `vfw.patch`.
JIRA issue: [CORE-15382](https://jira.reactos.org/browse/CORE-15382)

## TODO

- [x] Do tests.

### CODE REVIEW

File: dll/win32/msvidc32/msvideo1.c [hunk 1]
```diff
@@ -538,7 +538,11 @@ static LRESULT CRAM_GetInfo( const Msvideo1Context *info, ICINFO *icinfo, DWORD
 
     icinfo->dwSize = sizeof(ICINFO);
     icinfo->fccType = ICTYPE_VIDEO;
+#ifdef __REACTOS__
+    icinfo->fccHandler = info ? info->dwMagic : MSVC_MAGIC;
+#else
     icinfo->fccHandler = info ? info->dwMagic : CRAM_MAGIC;
+#endif
```
Comment (binarymaster) [inline dll/win32/msvidc32/msvideo1.c:545]: Please double-check whether this is correct change. A VfW codec could support several FOURCC's, but usually exposes just one for compression. (I'm pretty sure msvidc32 should expose `CRAM`)

Comment (katahiromz) [reply dll/win32/msvidc32/msvideo1.c]: Reverted.

Comment (katahiromz) [reply dll/win32/msvidc32/msvideo1.c]: Partially fixed.

File: dll/win32/msvidc32/msvideo1.c [hunk 2]
```diff
@@ -583,7 +583,11 @@ LRESULT WINAPI CRAM_DriverProc( DWORD_PTR dwDriverId, HDRVR hdrvr, UINT msg,
         if( info )
         {
             memset( info, 0, sizeof *info );
+#ifdef __REACTOS__
+            info->dwMagic = MSVC_MAGIC;
+#else
             info->dwMagic = CRAM_MAGIC;
+#endif
```
Comment (binarymaster) [inline dll/win32/msvidc32/msvideo1.c:590]: Note: You can attempt upstreaming this into Wine.

**Review by binarymaster — APPROVED** [review]
LGTM.

**Pull request merged**